recipes/gtsam/all/conanfile.py

```diff
@@ -161,3 +161,5 @@ def package(self):
 
     def package_info(self):
         self.cpp_info.libs = tools.collect_libs(self)
+        if self.settings.os == "Windows":
```

Comment:
```suggestion
        if self.settings.os == "Windows" and tools.Version(self.version) >= "4.0.3":
```

comes from https://github.com/borglab/gtsam/blob/4.0.3/gtsam/base/types.cpp, which didn't exist before 4.0.3

Reply:
True, thanks!